lib/tjbot.js

```diff
@@ -28,6 +28,7 @@ const colorToHex = require('colornames');
 const cm = require('color-model');
 const AudioContext = require('web-audio-api').AudioContext;
```

Comment:
We can remove the web-audo-api ...
I think I used it when I had the dance method in the library .. That has since been removed, web-audio-api can go.